docs/development/dev_howto.rst

```diff
@@ -569,12 +569,23 @@ As explained in the
 of the Astropy docs, there are (at least) two approaches for adding to the releases,
 each with pros and cons.
 
-We've had some pain due to merge conflicts in the releases notes and having to wait
-until the contributor rebases (and having to explain git rebase to new contributors).
+We recently chose to utilise `towncrier <https://towncrier.readthedocs.io/en/stable/>`__ for our release notes.
+
+- For each PR, a related 'fragment' file should be created in the ``docs/release-notes`` folder.
+- The naming convention of the file should be ``<PULL REQUEST NUMBER>.<TYPE>.rst``, where the available
```

Comment:
is the `type` necessary in the name? Can't it be interfered from the GitHub label?